packages/webview_flutter/webview_flutter/example/integration_test/webview_flutter_test.dart

```diff
@@ -1210,86 +462,6 @@ Future<void> main() async {
       await expectLater(controller.currentUrl(), completion(primaryUrl));
     },
   );
-
-  testWidgets(
-    'clearCache should clear local storage',
```

Comment:
Why are we losing this test?

Reply:
`clearCache` and `clearLocalStorage` are now two separate methods now. I'll add this test back and make it a test for just calling `clearLocalStorage` instead.